pages/linux/lvcreate.md

```diff
@@ -0,0 +1,19 @@
+# lvcreate
+
+> Creates a logical volume in an existing volume group.
```

Comment:
I added a short description of `VG`, please let me know if it is ok or I need to improve something.